autoload/coverage/gcov.vim

```diff
@@ -0,0 +1,134 @@
+" Copyright 2021 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+"{{{ Init
+
+let s:plugin = maktaba#plugin#Get('coverage')
+
+""
+" Directories under which to shallowly search for gcov data files.
+"
+" Temporary, to be removed in https://github.com/google/vim-coverage/issues/42
+let s:plugin.globals._gcov_temp_search_paths = ['.']
```

Comment:
Just noticed: These need to be in a guard like `has(globals, '_gcov_temp_search_paths')` so they won't overwrite existing values every time vim tries to source the file.

Reply:
Makes sense, good catch!